bucket/freeorion.json

```
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://freeorion.org/",
+    "description": "Turn-based space empire and galactic conquest (4X) game",
+    "version": "0.4.10.2",
+    "license": "GPL-2.0",
+    "url": "https://sourceforge.net/projects/freeorion/files/FreeOrion/FreeOrion%20Version%200.4.10/FreeOrion_v0.4.10.2_2021-08-01.f663dad_Win32_Setup.exe#/dl.zip",
+    "hash": "sha1:427bee2806be31138c8ddbb379e911f554f53479",
+    "bin": "FreeOrion.exe",
+    "shortcuts": [
+        [
+            "FreeOrion.exe",
+            "FreeOrion"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/freeorion/freeorion/releases/latest",
+        "regex": "FreeOrion_v([\\d.]+)_(?<file>[\\d\\-.a-z]+)_Win32_Setup.exe"
+    },
+    "autoupdate": {
+        "url": "https://sourceforge.net/projects/freeorion/files/FreeOrion/FreeOrion%20Version%20$majorVersion.$minorVersion.$patchVersion/FreeOrion_v$version_$matchFile_Win32_Setup.exe#/dl.zip"
+    },
+    "notes": "User data are stored in C:\\Users\\[USERNAME]\\AppData\\Roaming\\FreeOrion"
```

We don't add a note for the user data path unless the user *must* drop something in there for the game to work (e.g. proprietary game data).

```suggestion
    }
```